static_frame/core/yarn.py

```diff
@@ -64,23 +70,23 @@ class Yarn(ContainerBase, StoreClientMixin):
             '_deepcopy_from_bus',
             )
 
-    _series: Series
+    _series: TSeriesAny
```

Comment:
Why not `TSeriesObject`?

Reply:
Right! In my first pass accommodating generics, I erred on the side of over any-ing, again in large part because the number of changes needed was insane, so efficiency was key.